examples/videocomposer/tools/webvid_scripts/datacheck_webvid.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import json
+import os
+
+from tqdm import tqdm
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(description="WebVid Data Check")
+    parser.add_argument(
+        "--set", type=str, required=True, choices=["10M_train", "10M_val", "2M_train", "2M_val"], help="set name"
+    )
+    parser.add_argument("--part", type=int, help="part ID")
+    parser.add_argument("--root", type=str, required=True, help="dataset root directory")
+    return parser.parse_args()
+
+
```

Comment:
better to put the runnable script under `__main__` or `main()`, prevent the situation of auto running of the script when import this file